Dockerfile

```diff
@@ -34,6 +34,8 @@ COPY --from=BUILD_IMAGE /dist /usr/share/nginx/html
 ARG commit=local
 ENV COMMIT=$commit
 
+RUN docker login -u docker_hub_username -p docker_hub_pswd
```

Comment:
@directcuteo 
I am not sure about this in Dockerfile

Reply:
@bkontur I just put this a few minutes ago to test, but it's not working. I'm removing it